stor/obs.py

```diff
@@ -276,9 +280,9 @@ def __iter__(self):
     def _buffer(self):
         "Cached buffer of data read from or to be written to Object Storage"
         if self.mode in ('r', 'rb'):
-            return cStringIO.StringIO(self._path.read_object())
+            return io.StringIO(self._path.read_object())
```

Comment:
I think these should be `BytesIO`. Object Storage is storing binary blobs of data, not unicode.